github/github.go

```diff
@@ -0,0 +1,107 @@
+package github
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// Issue represents a GitHub issue
+type Issue struct {
+	// Unique id number for the GitHub issue
+	Number int
+
+	// Title for the GitHub issue
+	Title  string
+}
+
+// FetchIssues uses the GitHub command line application to fetch all open issues from the given repository
+func FetchIssues(repo string) ([]Issue, error) {
+	output, err := execIssueList(repo)
```

Comment:
It's a style thing but typically we don't factor these smaller functions out into their own functions if they're not used anywhere else. When reading this I now need to navigate to this function which is only 6 lines. In fact, your'e not saving any space because the error checking needs to be duplicated.

```diff
+	output, err := exec.Command("gh", "issue", "list", "--json", "number,title", "-R", repo).Output()
-	output, err := execIssueList(repo)
```

Sure it's a bit harder to read the first one, but you save the overhead of needing to navigate into that function which you're probably gonna need to do anyway to understand the code.